base/src/main/java/io/spine/type/KnownTypes.java

```diff
@@ -183,7 +187,8 @@ public TypeSet asTypeSet() {
     /**
      * Shows if the given {@link TypeUrl} is known the system.
      *
-     * @param typeUrl the {@link TypeUrl} to look up
+     * @param typeUrl
+     *         the {@link TypeUrl} to look up
```

Comment:
Let's substitute `{@link}` with `{@code}`, as in the generated Javadoc there is going to be a link to the parameter type anyway.